src/app/(simulation)/simulateur/[root]/page.tsx

```diff
@@ -49,7 +55,16 @@ export default function SimulateurPage() {
         isLoading={!isGuardInit || isGuardRedirecting}
       />
 
-      <SaveModal isOpen={isSaveModalOpen} closeModal={toggleSaveModal} />
+      <SaveModal
+        isOpen={isSaveModalOpen}
+        closeModal={toggleSaveModal}
+        mode="save"
+      />
+      <SaveModal
```

Comment:
On peut avoir un seul <Savemodal /> ici il me semble
```suggestion
      <SaveModal
              isOpen={isBackHomeModalOpen || isSaveModalOpen}
              closeModal={isSaveModalOpen ? toggleSaveModal : toggleBackHomeModal}
              mode={isSaveModalOpen ? "save" : "backHome"}
      />
```